src/entities/Map/routes/getMapPlacesMostActive.ts

```diff
@@ -0,0 +1,106 @@
+import { withAuthOptional } from "decentraland-gatsby/dist/entities/Auth/routes/withDecentralandAuth"
+import Context from "decentraland-gatsby/dist/entities/Route/wkc/context/Context"
+import ApiResponse from "decentraland-gatsby/dist/entities/Route/wkc/response/ApiResponse"
+import Router from "decentraland-gatsby/dist/entities/Route/wkc/routes/Router"
+import {
+  bool,
+  numeric,
+  oneOf,
+} from "decentraland-gatsby/dist/entities/Schema/utils"
+import { flat, sort } from "radash"
+
+import { getHotScenes } from "../../../modules/hotScenes"
+import { getSceneStats } from "../../../modules/sceneStats"
+import PlaceModel from "../../Place/model"
+import { FindWithAggregatesOptions, PlaceListOrderBy } from "../../Place/types"
+import { DEFAULT_MAX_LIMIT } from "../types"
+import { placesWithCoordinatesAggregates } from "../utils"
+import { validateGetPlaceListQuery } from "./getMapPlaces"
+
+export const getMapPlacesMostActive = Router.memo(
+  async (ctx: Context<{}, "url" | "request">) => {
+    const query = await validateGetPlaceListQuery({
+      positions: ctx.url.searchParams.getAll("positions"),
+      offset: ctx.url.searchParams.get("offset"),
+      limit: ctx.url.searchParams.get("limit"),
+      only_favorites: ctx.url.searchParams.get("only_favorites"),
+      only_featured: ctx.url.searchParams.get("only_featured"),
+      only_highlighted: ctx.url.searchParams.get("only_highlighted"),
+      order_by: PlaceListOrderBy.MOST_ACTIVE,
+      order:
+        oneOf(ctx.url.searchParams.get("order"), ["asc", "desc"]) || "desc",
+      with_realms_detail: ctx.url.searchParams.get("with_realms_detail"),
+      search: ctx.url.searchParams.get("search"),
+      categories: ctx.url.searchParams.getAll("categories"),
+    })
+
+    const sceneStats = await getSceneStats()
+
+    const hotScenes = getHotScenes()
```

Comment:
This is a mix of environments. getHotScenes was defined in modules, that's part of the frontend, but entities are part of the backend. Would you mind adding a similar function inside utils?

Reply:
updated!